runtime/compiler/optimizer/VectorAPIExpansion.cpp

```diff
@@ -1496,6 +1515,13 @@ TR::Node *TR_VectorAPIExpansion::broadcastCoercedIntrinsicHandler(TR_VectorAPIEx
       TR::ILOpCodes splatsOpCode = TR::ILOpCode::createVectorOpCode(OMR::vsplats, elementType.scalarToVector(OMR::DataType::bitsToVectorLength(vectorLength)));
 
       TR::Node::recreate(node, splatsOpCode);
+
+      if (TR::Options::getVerboseOption(TR_VerboseVectorAPI))
+         {
+         TR::ILOpCode opcode(splatsOpCode);
+         TR_VerboseLog::writeLine(TR_Vlog_VECTOR_API, "Vectorized using %s%s in %s", opcode.getName(),
```

Comment:
Minor nitpick, but is there a reason we have different string for Scalarized broadcast vs Vectorized case?

Reply:
Good question. Scalarized broadcast does not have any opcode. It just clones its child `numLanes` number of times. Vectorized broadcast will produce `vsplats` and we need to print the actual vector type for that `vsplats` (vector operation).